tests/acceptance/30_generic_methods/network/file_download.cf

```diff
@@ -0,0 +1,57 @@
+#######################################################
+#
+# Test checking if a file can be downloaded from a http
+#
+#######################################################
+
+bundle common acc_path
+{
+  vars:
+    "root" string => getenv("NCF_TESTS_ACCEPTANCE", 1024);
+}
+
+body common control
+{
+      inputs => { "${acc_path.root}/default.cf.sub", "${acc_path.root}/default_ncf.cf.sub", "@{ncf_inputs.default_files}" };
+      bundlesequence  => { configuration, default("${this.promise_filename}") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+{
+  vars:
+
+    "tmp"                    string => getenv("TEMP", 1024);
+
+    "source_file"            string => "http://www.normation.com/robots.txt";
+    "destination_file"       string => "${tmp}/test_file_download.txt";
```

Comment:
I would also like a test where the file is already there, so don't need to download it